WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```diff
@@ -259,7 +262,10 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
         if (savedInstanceState == null) {
-            if (mIsEmailSignup) {
+             // Start loading reader tags so they will be available asap
+             ReaderUpdateServiceStarter.startService(getActivity(), EnumSet.of(ReaderUpdateLogic.UpdateTask.TAGS));
+
+             if (mIsEmailSignup) {
```

Comment:
```suggestion
            // Start loading reader tags so they will be available asap
            ReaderUpdateServiceStarter.startService(getActivity(), EnumSet.of(ReaderUpdateLogic.UpdateTask.TAGS));

            if (mIsEmailSignup) {
```

These added lines have an extra space in their indention.  Can we update them to be a factor of four spaces for code style consistency?